integration-tests/jest/jest-utils.js

```diff
@@ -0,0 +1,21 @@
+const { screenshotDir } = require('../utils/integration-config')
+
+
+const defaultTimeout = 5 * 60 * 1000
+
+const withGlobalJestPuppeteerContext = test => async () => await test({ context, page: await context.newPage() })
```

Comment:
This `async` and `await` look unnecessary. The test function will already return a promise.